_docker-library/docs/dart/content.md

```diff
@@ -1 +1,18 @@
-TODO: For example, see https://github.com/docker-library/docs/tree/master/node
+# What is Dart?
+
+Dart is a client-optimized language for developing fast apps on any platform.
+Its goal is to offer the most productive programming language for multi-platform
+development, paired with a flexible execution runtime platform for app
+frameworks. For more details, see https://dart.dev.
+
+By utilizing Dart's support for ahead-of-time (AOT)
+[compilation to executables], you can create very small runtime images (~10 MB).
+
+%%LOGO%%
```

Comment:
?? Is this new syntax? 😁

Reply:
This came from https://github.com/docker-library/docs/tree/master/node; I suspect Docker does some rewriting of this content to insert the logo there

Reply:
👍